### ISSUE TITLE
Add a way to select the Q range to fit

### ISSUE THREAD

**Issue by mdoucet** [reflectometry/refl1d#178]
When using the model builder and loading a data set into the model, there doesn't seem to be a way to change the Q range used for the fit. I could see that being useful even for models loaded as scripts so that you don't have to code it in.

**Comment by bmaranville** [comment]
Could this be addressed with masking during reduction instead? That seems like a more natural place to do it, to me.

**Comment by mdoucet** [comment]
Good point. Sometimes I do this to explore potential issues, or to just fit the scale under the critical edge.
